### ISSUE TITLE
Possible way to handle with increase in UI extension addons

### ISSUE THREAD

**Issue by flatsiedatsie** [WebThingsIO/gateway#2764]
Just a quick rough experiment. The idea was to see if the menu could be more like an app tray. I think it could.

![Screenshot 2021-01-30 at 13 05 45](https://user-images.githubusercontent.com/805405/106355984-22cbe300-62fc-11eb-8a38-791382b4977c.png)

**Comment by kgiori** [comment]
Conceptually it seems valid since you have have created so many extension add-ons (bravo). Cosmetically I would suggest making the width of the columns narrower if possible.

**Comment by flatsiedatsie** [comment]
Agreed. But this was really just a two-minute CSS hacking experiment, to see if it was possible/easy to go the app-tray route at all.
